src/tools/main.ts

```diff
@@ -16,18 +12,11 @@ import Config from './config';
 
 export default async function main(specFiles, c) {
   const config = new Config(c);
-  const filePaths = getPreparedSpecFilePaths(specFiles, config.tags);
-
-  await Promise.all(
-    filePaths.map((file) =>
-      readSpecFilePromise(file, {
-        dereference: true,
-      }),
-    ),
-  )
-    .then((schemas) => {
-      const spec = _.merge(...schemas);
+  const spec = getPreparedSpec(specFiles, config.tags);
+  const copiedSpec = JSON.stringify(spec); // dereferenceが内部状態を変えてしまうためcopy
```

Comment:
@MtBlue81 細かい話ですが、個人的にはcopiedSpecはこっちの方がいいかもと思いました。
```suggestion
  // dereferenceが内部状態を変えてしまうためcopy
  const copiedSpec = JSON.parse(JSON.stringify(spec));
```

Reply:
あぁ。なるほど、たしかに 👍